jpa/eclipselink.jpa.test.jse/src/it/java/org/eclipse/persistence/jpa/test/mapping/model/ChildMultitenant.java

```diff
@@ -31,7 +31,7 @@
 @Entity
 @Table(name = "children")
 @Multitenant(MultitenantType.TABLE_PER_TENANT)
-@TenantTableDiscriminator(type = TenantTableDiscriminatorType.SCHEMA)
+@TenantTableDiscriminator(type = TenantTableDiscriminatorType.PREFIX)
```

Comment:
this changes meaning of the test

Reply:
It is reverted back to the origin with `TenantTableDiscriminatorType.SCHEMA`, but limited to MySQL DB only plus `DROP SCHEMA` command added to ensure, that repeated test executions against same DB will not fail.